### PR TITLE
Simple edit to Garbage collection arabic translation

### PR DESCRIPTION
**Pull request by Ahmed-Adel3**

### CODE REVIEW

File: 1-js/03-code-quality/02-coding-style/1-style-errors/solution.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-function pow(x,n)  // <- no space between arguments
-{  // <- figure bracket on a separate line
-  let result=1;   // <- no spaces before or after =
-  for(let i=0;i<n;i++) {result*=x;}   // <- no spaces
-  // the contents of { ... } should be on a new line
+function pow(x,n)  // <- لا مسافات بين المعطيات
+{  // <- القوس المعقوف في سطر جديد وحده
+  let result=1;   // <- لا مسافة قبل = 
+  for(let i=0;i<n;i++) {result*=x;}   // <- لا مسافة
+  // محتوي { ... } يجب ان يكون علي سطر جديد
```
[the remark this answers is not in the document]
Comment (3imed-jaberi) [reply 1-js/03-code-quality/02-coding-style/1-style-errors/solution.md:9]: @Ahmed-Adel3, please avoid mark anything as resolvd with out explained why or make the change.

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 2]
```diff
@@ -333,16 +334,16 @@ Here's an example of an `.eslintrc` file:
 }
 ```
 
-Here the directive `"extends"` denotes that the configuration is based on the "eslint:recommended" set of settings. After that, we specify our own.
+هنا التوجيه `"extends"` يشير إلى أن التكوين يستند إلى مجموعة اعدادات "eslint:recommended" . بعد ذلك يمكننا تحديد الاعدادات الخاصة بنا.
```
[the remark this answers is not in the document]
Comment (OmarioHasan) [reply 1-js/03-code-quality/02-coding-style/article.md:337]: agree with you

[the remark this answers is not in the document]
Comment (OmarioHasan) [reply 1-js/03-code-quality/02-coding-style/article.md:337]: إعدادات

File: 1-js/03-code-quality/02-coding-style/article.md [hunk 3]
```diff
@@ -104,20 +104,19 @@ if (
...
-- **Horizontal indents: 2 or 4 spaces.**
+- **المسافات البادئة الأفقية: 2 أو 4 مسافات.**
 
-    A horizontal indentation is made using either 2 or 4 spaces or the horizontal tab symbol (key `key:Tab`). Which one to choose is an old holy war. Spaces are more common nowadays.
+   يتم عمل مسافة بادئة أفقية باستخدام مسافتين أو 4 مسافات أو رمز علامة التبويب الأفقية (key `key:Tab`). أيهما كان اختيارك فهي حرب قديمة. المساحات هي الأكثر شيوعًا في الوقت الحاضر.
 
-    One advantage of spaces over tabs is that spaces allow more flexible configurations of indents than the tab symbol.
-
-    For instance, we can align the arguments with the opening bracket, like this:
+  هنالك ميزة واحدة للمسافات عن علامات التبويب الا و هي أن المسافات تسمح بتكوينات مسافات بادئة أكثر مرونة من رمز علامة التبويب.
```
Comment (3imed-jaberi) [inline 1-js/03-code-quality/02-coding-style/article.md:117]:
```suggestion
  هنالك ميزة واحدة للمسافات عن علامات التبويب الا و هي أن المسافات تسمح بإعدادات مسافات بادئة أكثر مرونة من رمز علامة التبويب.
```

**Pull request merged**